### PR TITLE
Move Create/Join Textbox to Home Page

### PR DESCRIPTION
**Pull request by daltonfury42**
image

### CODE REVIEW

File: simplq/src/components/pages/Home/CreateJoinForm.jsx [hunk 1]
```diff
@@ -52,15 +50,21 @@ const CreateQueue = ({ history }) => {
...
-          <CreateQButton onClick={() => handleClick(textFieldValue)} />
-        )}
+      <div className={styles['button-group']}>
+        <div>
+          {createInProgress ? (
+            <LoadingIndicator />
+          ) : (
+            <CreateQButton onClick={() => handleClick(textFieldValue)} />
+          )}
+        </div>
```
Comment (maaverik) [inline simplq/src/components/pages/Home/CreateJoinForm.jsx:60]: handleClick doesn't take arguments

Comment (daltonfury42) [reply simplq/src/components/pages/Home/CreateJoinForm.jsx:60]: Done

File: simplq/src/components/pages/Home/index.jsx [hunk 2]
```diff
@@ -1,10 +1,26 @@
...
+        <CreateJoinForm history={props.history} />
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320">
+          <path
+            fill="#6C63FF29"
+            fillOpacity="1"
+            d="M0,224L80,192C160,160,320,96,480,101.3C640,107,800,181,960,181.3C1120,181,1280,107,1360,69.3L1440,32L1440,320L1360,320C1280,320,1120,320,960,320C800,320,640,320,480,320C320,320,160,320,80,320L0,320Z"
+          />
+        </svg>
+      </div>
       <BenefitsInfo />
```
Comment (maaverik) [inline simplq/src/components/pages/Home/index.jsx:24]: Could you move this landing page div into another component and import it here? I guess it doesn't make sense to put it in staticInfos anymore. I feel it looks ugly the way it is right now.

File: simplq/src/components/Routes.jsx [hunk 3]
```diff
@@ -14,10 +13,8 @@ export default () => {
       <ErrorBoundary>
         <Switch>
           <Route path="/" exact component={Home} />
-          <Route path="/create" exact component={CreateQueue} />
           <Route path="/queue/:queueId" exact component={AdminQueue} />
           <Route path="/j/:queueId" exact component={JoinQueueWithDetails} />
-          <Route path="/join" exact component={JoinQueueWithLink} />
           <Route path="/token/:tokenId" exact component={QueueStatus} />
```
Comment (maaverik) [inline simplq/src/components/Routes.jsx:18]: If joinQueueWithLink is not needed anymore, could you remove it?

File: simplq/src/components/pages/Home/index.jsx (hunk 2 above)
Comment (daltonfury42) [reply simplq/src/components/pages/Home/index.jsx:24]: It's not in staticInfos anymore. I moved it into the index.jsx file. If you feel we need to create a separate component for the landing page (a separate page) and move it out of index.jsx, I can. What do you say?

File: simplq/src/components/Routes.jsx (hunk 3 above)
Comment (daltonfury42) [reply simplq/src/components/Routes.jsx:18]: I've just now removed it in the next PR, where I have implemented the join queue button click

Comment (daltonfury42) [reply simplq/src/components/Routes.jsx:18]: Actually you are right, a separate component will be a good idea. I moved it out

https://github.com/SimplQ/simplQ-frontend/pull/218/commits/d7d953c83795a7d3c4e797aab8b04a69c37cf8c4

File: simplq/src/components/pages/Home/index.jsx (hunk 2 above)
Comment (daltonfury42) [reply simplq/src/components/pages/Home/index.jsx:9]: Actually you are right, a separate component will be a good idea. I moved it out

https://github.com/SimplQ/simplQ-frontend/commit/d7d953c83795a7d3c4e797aab8b04a69c37cf8c4

File: simplq/src/components/pages/Home/CreateJoinForm.jsx [hunk 4]
```diff
@@ -38,29 +37,34 @@ const CreateQueue = ({ history }) => {
...
+        <div>
+          {createInProgress ? (
+            <LoadingIndicator />
+          ) : (
+            <CreateQButton onClick={() => handleClick()} />
+          )}
+        </div>
+        <div>
+          <JoinQButton onClick={() => history.push('/join/')} />
+          {/* FIXME */}
```
Comment (daltonfury42) [inline simplq/src/components/pages/Home/CreateJoinForm.jsx:63]: Raised #219 to handle this logic

**Review by maaverik — APPROVED** [review]

**Pull request merged**